lib/cli/parse.js

```diff
@@ -191,6 +191,10 @@ function nodemonOption(options, arg, eatNext) {
     options.stdin = false;
   }
 
+  else if (arg === '--no-startup' || arg === '-S') {
+    options.runOnStartup = false;
```

Comment:
Sorry, just code reviewing now - this property should be named the same as the CLI flag, so it should be `noStartup` - but then I'm not sure if it should be true or false...

Should it be something like "execute on change", like `execOnChange` and `--exec-on-change`. Or `--no-immediate-start`...hmm, doesn't sound right either.

Overall the change is solid, tests included, but the flag naming needs to be consistent with the CLI and I'd like the naming to clear what it does.

Maybe use this issue to run through some options, then change the code once it's all settled?